animeal/res/en.lproj/Localizable.strings

```diff
@@ -39,6 +39,7 @@
 "more.about"                                      = "About (terms and conditions)";
 "more.account"                                    = "Account";
 "more.aboutShort"                                 = "About";
+"more.aboutContent"                               = "Who are we?\nAnimal Project was founded in 2020 by animal lovers with different professions and experiences. For decades, our team members  have been independently helping strays and raising awareness through personal channels.\n\nWhat is our goal?\nWe exist so that dogs and cats no longer have to live in the streets of Georgia. Our goal is to create a habitable, comfortable environment for companion animals and raise awareness within our communities.\n\nHow are we working on this goal?\nOur team heavily relies on international best practices and experience. Our areas of work include:\n· Raising social awareness;\n· Advocating with the government;\n· Charitable and humanitarian projects in collaboration with the private sector, donor organizations, and local communities;";
```

Comment:
Is the project name "Animeal" OR it's "Animal".
Also in some places we have more than one space.
Lastly, in general we end our sentences with dots and we have used ';' instead. is this intended?

Reply:
Animeal is app name, Animal - is the name of volunteer's project. So it's a bit messy but it is different things :)